.github/workflows/pull_request_label.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       pull-requests: write
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/labeler@4f052778de9a9b80cb16cfb9079b02287285a4cb
+      - uses: actions/labeler@ac9175f8a1f3625fd0d4fb234536d26811351594
```

Comment:
The blast tool bumped this version to a pre-release; this reverts to the latest stable version of the action.

https://github.com/dart-lang/ecosystem/issues/108